tests/e2e/pages/bounty/bounty.js

```diff
@@ -70,23 +76,30 @@ export function CreatePullRequest() {
   ).as("getIssuePrData");
 
   bounty.getCreatePullRequestButton().click({ force: true });
+  cy.wait(["@gqlBranchesQuery", "@gqlRepositoriesQuery"]);
   cy.get("h3").contains("Create Pull Request").should("exist");
+  cy.wait(1500);
   cy.get(".react-select__input").click({ force: true }).type("{enter}");
   cy.get("span").contains("Create Pull Request").click({ force: true });
-  cy.wait("@postPullRequest");
-  cy.wait([
-    "@getIssueCommentsGithub",
-    "@getGithubCurrentRepos",
-    "@getIssuePrData",
-    "@getPrCount",
-    "@getPrGithub",
-  ]);
+  cy.wait(500);
 
+  cy.intercept("POST", "/api/pull-request/", (req) => {
+    req.redirect("/api/pull-request", 200);
```

Comment:
?! Why don't we simply intercept `/api/pull-request/` instead of intercepting this, redirecting, and then intercepting the former?